daemon.lua

```diff
@@ -0,0 +1,58 @@
+--
+-- Copyright (C) 2024 Ring Zero Desenvolvimento de Software LTDA.
+--
+-- Permission is hereby granted, free of charge, to any person obtaining
+-- a copy of this software and associated documentation files (the
+-- "Software"), to deal in the Software without restriction, including
+-- without limitation the rights to use, copy, modify, merge, publish,
+-- distribute, sublicense, and/or sell copies of the Software, and to
+-- permit persons to whom the Software is furnished to do so, subject to
+-- the following conditions:
+--
+-- The above copyright notice and this permission notice shall be
+-- included in all copies or substantial portions of the Software.
+--
+-- THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+-- EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+-- MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+-- IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY
+-- CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+-- TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+-- SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+--
+
+local thread  = require("thread")
+local socket  = require("socket")
+local inet    = require("socket.inet")
+local linux   = require("linux")
+local rcu     = require("rcu")
+local mailbox = require("mailbox")
+local filter  = require("dome/filter")
+
+local shouldstop = thread.shouldstop
+local sock = socket.sock
+
+local telegraf = inet.tcp()
+telegraf:connect(inet.localhost, 8094)
```

Comment:
`connect` e `send` levantam erro? se nao, falhar silenciosamente aqui eh ruim.

Reply:
eu criei um ticket para tratar o a tentativa de reconexao.. hj o `socket` levanta erro exceto no caso do read, para tratarmos o `EAGAIN` no caso de async.. uma ideia seria passar a API toda para error cond.. em vez de levantar excessao.. o que acha?

Reply:
minha preocupacao eh mais quem ta levantando o serviço ver que nao conectou. tipo, eu rodo o daemon, como eu sei se conectou ou nao.

Reply:
problema maior ta na reconexao.. se o endpoint cair por exemplo.. qnt a levantar erro ou retornar a condret, queria sua opiniao msm.. msm se for none/dont care ;-)